### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by JAlvarezJarreta**
I have found these whilst working on the release. I think these changes in the documentation will help newcomers.

I have also commented out two unused libraries.

### CODE REVIEW

File: modules/Bio/EnsEMBL/Compara/DBSQL/DnaFragAdaptor.pm [hunk 1]
```diff
@@ -207,13 +207,6 @@ sub fetch_all_by_GenomeDB_region {  # DEPRECATED
   Arg [-CODON_TABLE_ID] (opt)
              : Integer. Only returns DnaFrags that are translated with this codon table
   Example    : $human_scaffolds = $dnafrag_adaptor->fetch_all_by_GenomeDB($human_gdb, -COORD_SYSTEM_NAME => 'scaffold', -CELLULAR_COMPONENT => 'NUC', );
-  Arg [2]    : (optional) string $coord_system_name
-  Arg [3]    : (optional) string $name
-  Arg [4]    : (optional) boolean $is_reference
-  Arg [5]    : (optional) string $cellular_component
-  Example    : my $human_chr_dnafrags = $dnafrag_adaptor->
-                   fetch_all_by_GenomeDB_region(
-                     $human_genome_db, 'chromosome')
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/DBSQL/DnaFragAdaptor.pm]: This is a leftover of the previous method, `fetch_all_by_GenomeDB_region`, which, BTW was listed as to be removed in e98. Can you remove it in this PR too ? And update DEPRECATED.md at the same time

Comment (JAlvarezJarreta) [reply modules/Bio/EnsEMBL/Compara/DBSQL/DnaFragAdaptor.pm]: Done. I have also found a `subtest` for the deprecated method and I have removed it.

File: modules/Bio/EnsEMBL/Compara/DBSQL/DnaFragAdaptor.pm [hunk 2]
```diff
@@ -44,8 +44,7 @@ Bio::EnsEMBL::Compara::DBSQL::DnaFragAdaptor
   
   $dnafrag = $dnafrag_adaptor->fetch_by_dbID(905406);
   $dnafrag = $dnafrag_adaptor->fetch_by_GenomeDB_and_name($human_genome_db, 'X');
-  $dnafrags = $dnafrag_adaptor->fetch_all_by_GenomeDB_region(
-                                $human_genome_db, 'chromosome')
+  $dnafrags = $dnafrag_adaptor->fetch_all_by_GenomeDB($human_genome_db)
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/DBSQL/DnaFragAdaptor.pm]: Add `-COORD_SYSTEM_NAME => 'chromosome'` to preserve the original intent

File: modules/t/dnaFragAdaptor.t [hunk 3]
```diff
@@ -104,28 +104,6 @@ subtest "Test Bio::EnsEMBL::Compara::DBSQL::DnaFragAdaptor::fetch_by_GenomeDB_an
     done_testing();
 };
 
-subtest "Test Bio::EnsEMBL::Compara::DBSQL::DnaFragAdaptor::fetch_all_by_GenomeDB_region method", sub {
```
Comment (muffato) [inline modules/t/dnaFragAdaptor.t]: Actually, the test itself has already been adapted to use `fetch_all_by_GenomeDB`. It's only the title of the subtest that needs change, but the test can stay

**Pull request merged**